### ISSUE TITLE
compare to CvT

### ISSUE THREAD

**Issue by liyunsheng13** [yhlleo/VTs-Drloc#1]
Hi

Thanks for sharing this good work. I'm curious about why the proposed loss function can outperform CvT, which contains a depthwise convolution that is capable to learn local features.

**Comment by yhlleo** [comment]
Hi @liyunsheng13, 

Good question. It's also similar on the results of ResNet, where there are only convolution layers. We guess that the proposed loss acts as a regularizer, which helps both VTs and CNNs learn local features better, especially in the earlier epochs. You're right the convolutional layers are capable to learn local features. In our experiments, we can see only marginally or the same performance with longer training.

**Comment by yhlleo** [comment]
Besides, if you in our supplementary materials, we also show a figure about the training with different VTs on CIFAR-100.
![epochs](https://user-images.githubusercontent.com/13311630/143979290-5db6fd68-ab94-41eb-b6be-7c4cec8791c4.png)

It shows that the gain of our loss for the architecture with depthwise convolution is obviously less.

**Comment by liyunsheng13** [comment]
Got it. Thanks for your response.
